ambry-api/src/main/java/com.github.ambry/account/Account.java

```diff
@@ -278,6 +300,9 @@ public boolean equals(Object o) {
     if (status != account.status) {
       return false;
     }
+    if (snapshotVersion != account.snapshotVersion) {
```

Comment:
I'm not sure it is a good idea to include this in the equals - two accounts are equal by virtue of their properties - not on their snapshot versions, correct?